src/services/dealer.ts

```diff
@@ -318,3 +319,23 @@ export const putUserAccount = async ({
     return handleValidationError(error, { swallowErrors: true })
   }
 }
+
+export const deleteUser = async ({
```

Comment:
can you move this to the user service please? on our side, it's currently on the dealer service, which is not only off with backends structure, but also with our own now that we have non-dealer users

https://dealer-service.preprod.carforyou.ch/swagger-ui/index.html#/User/deleteUserUsingDELETE

Reply:
it will be moved to the dealer service https://github.com/carforyou/carforyou-api-client-pkg/pull/351#discussion_r722128813